etc/kube/pachyderm.json

```diff
@@ -1,6 +1,4 @@
 {
```

Comment:
Most of the changes to this file are superficial due to json encodings not being deterministic.